dspace-rest/src/main/java/org/dspace/rest/ItemsResource.java

```diff
@@ -992,7 +992,7 @@ else if (!AuthorizeManager.authorizeActionBoolean(context, bitstream, org.dspace
             {
                 sql += "QUALIFIER='" + metadata[2] + "' AND ";
             }
-            sql += "dbms_lob.substr(TEXT_VALUE, 40)='" + metadataEntry.getValue() + "' AND ";
+            sql += "TEXT_VALUE='" + metadataEntry.getValue() + "' AND ";
```

Comment:
I think that is still a problem with error ORA-00932 (direct compare CLOB and string - see above TODO comment) for oracle db.